torch.hash.yml

```diff
@@ -0,0 +1,112 @@
+yamls/us/ceremony_data.yml:
+  hash: 394bdc06637b9a166232aa16ef5d8764929bbdf6
+  extracted:
+    Header: true
```

Comment:
Remove this file. If you have a second, can you please add this file to the gitignore?